sources/osgUtil/osgUtil.js

```diff
@@ -1,10 +1,12 @@
 'use strict';
 var Composer = require( 'osgUtil/Composer' );
+var Debug = require( 'osgUtil/debug' );
```

Comment:
should be var debug =  and not Debug.
First upper letter is for class in the code too